docs/core/integration-quality-scale/rules/iqs043.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "IQS043 - The documentation provides removal instructions"
+---
+
+IQS043 is as follows:
+
+The documentation provides removal instructions
+
+## Reasoning
+
+Removing a device or service from Home Assistant isn't always straightforward.
+The documentation should provide clear instructions on how to remove it, even if it's as simple as "go the integrations page and click delete".
```

Comment:
I suggest we remove this example as it's not complete. You can't click delete on the integrations page. To make it complete it would just be a repetition of the actual example below.
```suggestion
The documentation should provide clear instructions on how to remove the device or service.
```